common.blocks/dropdown/_switcher/dropdown_switcher_link.js

```diff
@@ -11,7 +11,7 @@ modules.define('dropdown', ['link'], function(provide, _, Dropdown) {
  */
 provide(Dropdown.decl({ modName : 'switcher', modVal : 'link' }, null, /** @lends dropdown */{
     live : function() {
-        this.liveInitOnBlockInsideEvent('click', 'link', this.onSwitcherClick);
+        this.liveInitOnBlockEvent('click', 'link', this.onSwitcherClick);
```

Comment:
This part is not that obvious. As far as I remember it was made this way not but the mistake. One must check how clear this API will be from the pov of block inheritance